assets/app/features/upgrade/controllers/upgrade7-openstack-services.controller.js

```diff
@@ -48,19 +51,16 @@
                     
                 ).finally(function() {
                     // Either on sucess or failure, the openStackServices has been completed.
-                    vm.openStackServices.completed = true;
-
-                    // Update openStackServices validity
-                    
-
+                    vm.openStackServices.completed = true;              
+                    vm.openStackServices.running = false;
```

Comment:
I kind of understand this mistake. Checking `suseLazySpinner` directive, there is none documentation at all regarding how it's intended to be used or what their parameters meant to be. 
I've created #57 to address it